### ISSUE TITLE
Keyboard navigation for UI

### ISSUE THREAD

**Issue by rbreeze** [argoproj/argo-cd#5466]
# Summary

Easy navigation of UI with the keyboard. 

# Motivation

Better accessibility, and productivity with keyboard shortcuts.

# Proposal

### Application List: 
- Navigate applications with arrow keys
- `/` to search apps
- `Enter` to go to selected app

### Application Details
- Navigate resource tree or pod view with arrow keys
- `Enter` to open details panel for selected resource
- `l` key to open logs for selected pod
- `d` key to open app details panel
- `t` key to open app diff
- `ctrl+R` to refresh application

There are many others that can/should be implemented, the above is a first pass - and community suggestions for keybindings are encouraged

**Comment by reginapizza** [comment]
Kind of related to #4829 I think, or more that that issue relates to this as a whole

**Comment by jsoref** [comment]
Probably worth looking at the https://github.com/derailed/k9s/ keybindings.

I'm not a fan of all of them, but...
